scripts/release.sh

```diff
@@ -19,11 +19,6 @@ if [[ ! "${NAME}" =~ ^[a-zA-Z0-9_]+$ ]]; then
   exit 1
 fi
 
```

Comment:
This was removed when I was sourcing this script for the initial creation of the runner; it didn't need to be and can be reverted